README.md

```diff
@@ -25,6 +25,18 @@ The Presto Foundation maintains a [public calendar](https://calendar.google.com/
 
 ## Members
 
-The TSC voting members are initially the [Project’s Committers](https://github.com/prestodb/presto/wiki/committers), and the current TSC chair is Nezih Yigitbasi ([@nezihyigitbasi](https://github.com/nezihyigitbasi)). At the inception of the project, the Committers of the Project will be as set forth within the "CONTRIBUTING" file within the Project’s code repository
+The TSC voting members are initially the [Project’s Committers](https://github.com/prestodb/presto/wiki/committers).
```

Comment:
can we no longer use the wiki page for maintainers

1) can we add affiliations
2) can we add term lengths/dates

Reply:
Yes, good point. I think it makes sense to maintain the list of committers in CONTRIBUTING.md in .github, so it replicates across all repos, and so were only tracking that info in one place. I'll fix this link.

I can also add term lengths and affiliations for the TSC members in this file (even though they're initially the same).

Reply:
I've just pushed this in 62028e2.  A subsequent commit (5618251) contains a proposal for staggering term lengths to encourage continuity during and after elections.